public/locales/en/settings.json

```diff
@@ -3,6 +3,16 @@
   "save": "Save",
   "saving": "Saving...",
   "reset": "Reset",
+  "language": "Language",
```

Comment:
Would it be helpful to clarify what is the target of language picker? 
For example:
```suggestion
  "language": "Web UI Language",
```

Reply:
I don't think it's necessary. It's expected to change the Web UI language, as we are in the Web UI settings no? 🤔

Reply:
I think `Language` is ok.